lib/src/mt_main.c

```diff
@@ -489,7 +489,7 @@ mtl_handle mtl_init(struct mtl_init_params* p) {
        impl->user_tx_queues_cnt, impl->user_rx_queues_cnt,
        mt_get_user_params(impl)->flags);
   /* parse rss config, not use now */
-  impl->rss_mode = MT_RSS_MODE_NONE;
+  impl->rss_mode = p->rss_mode;
```

Comment:
We should add a auto detect(check if rte_flow capability) if user not assigned rss mode. Fine for current PR now, we can enhance in later commits.

Reply:
> We should add a auto detect(check if rte_flow capability) if user not assigned rss mode. Fine for current PR now, we can enhance in later commits.

noted